src/utils/FixedQueue.ts

```diff
@@ -0,0 +1,55 @@
+import * as _ from 'underscore';
```

Comment:
The more I look at it, the more I think it simply adds complexity for its use. @MichelML's arguments are excellent.